### PR TITLE
Fixes bug 1305616 - Stop storing memory_report into Elasticsearch.

### PR DESCRIPTION
**Pull request by adngdb**
@lonnen this is more of a political decision than a code one, so I'm asking you for review.

### CODE REVIEW

File: socorro/external/es/crashstorage.py [hunk 1]
```diff
@@ -231,9 +231,12 @@ class ESCrashStorageRedactedJsonDump(ESCrashStorageRedactedSave):
     required_config.es_redactor.forbidden_keys = change_default(
         Redactor,
         "forbidden_keys",
-        "upload_file_minidump_flash1.json_dump, "
-        "upload_file_minidump_flash2.json_dump, "
-        "upload_file_minidump_browser.json_dump"
+        (
+            "memory_report, "
```
Comment (peterbe) [inline socorro/external/es/crashstorage.py]: If this is set in consulate will this new default not take effect?

File: socorro/unittest/external/es/test_crashstorage.py [hunk 2]
```diff
@@ -464,6 +469,7 @@ def test_success_with_limited_json_dump_class(self, espy_mock):
             k: a_processed_crash['json_dump'][k]
             for k in modified_config.json_dump_whitelist_keys
         }
+        del expected_processed_crash['memory_report']
```
Comment (peterbe) [inline socorro/unittest/external/es/test_crashstorage.py]: I don't understand what the point of this is

File: socorro/unittest/external/es/test_crashstorage.py [hunk 3]
```diff
@@ -225,7 +229,7 @@ def test_bulk_index_crash(self):
                 x,
                 a_processed_crash['uuid'][2:]
             )
-            # Ensure that the document was indexed by attempting to retreive it.
+            # Ensure that the document was indexed by attempting to retreive it
```
Comment (peterbe) [inline socorro/unittest/external/es/test_crashstorage.py]: Shouldn't there be a test that tries to retrieve it afterwards that checks that `memory_report` did NOT get stored.

File: socorro/unittest/external/es/test_crashstorage.py (hunk 2 above)
Comment (adngdb) [inline socorro/unittest/external/es/test_crashstorage.py]: That is the answer to your previous question. It's how I verify that the ``memory_report`` has not been saved in the database. A few lines below there is a comparison that what has been sent to ES is the same as ``expected_processed_crash``.

Comment (peterbe) [inline socorro/unittest/external/es/test_crashstorage.py]: Ok. I think I get it.

**Pull request merged**